internal/modifiers/timemodifier/replace_if_outdated.go

```diff
@@ -5,11 +5,12 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-func ReplaceIfOutdated() tfsdk.AttributePlanModifier {
+func ReplaceIfOutdated() planmodifier.String {
```

Comment:
I believe this now can be replaced with simpler `stringplanmodifier.RequiresReplaceIf()` usage since the new framework logic no longer performs the `Computed` check. Otherwise, we may want to at least consider removing the `tfsdk.ValueAs()` logic, since `planmodifier.StringRequest.StateValue` will already be a `types.String` 👍